src/main/res/values-de/strings_pref.xml

```diff
@@ -8,6 +8,8 @@
     <string name="pref_confirm_on_exit_desc">Zeigt einen Dialog bevor Locus mit dem Zurück-Knopf beendet wird</string>
     <string name="pref_double_click">Doppelklicken</string>
     <string name="pref_double_click_desc">ZURÜCK doppelklicken um das Spiel zu beenden</string>
+    <string name="pref_run_screen_off">App läuft bei ausgeschaltetem Display</string>
+    <string name="pref_run_screen_off_desc">Stellt sicher, dass die App auch wenn das Display ausgeschaltet ist läuft.</string>
```

Comment:
Better grammer:
Stellt sicher, dass die App auch dann läuft, wenn das Display ausgeschaltet ist.

Reply:
Typo and grammer have been fixed.